mobile/src/reducers/feeds/feeds.epics.ts

```diff
@@ -13,9 +13,15 @@ import { Alert } from 'react-native';
 import { retryWhen, map } from 'rxjs/operators';
 import { UnaryFunction } from 'rxjs/interfaces';
 import { handleHttpErrors } from '../errors/errors.epics';
+import { LoadUserEmployeeFinished } from '../user/user.action';
 
 export const pagingPeriodDays = 10;
 
+export const loadUserEmployeeFinishedEpic$ = (action$: ActionsObservable<LoadUserEmployeeFinished>, state: AppState, deps: DependenciesContainer) =>
+    action$.ofType('LOAD-USER-EMPLOYEE-FINISHED')
```

Comment:
ah, ok, you do it here... but why do you have this call in componentDidMount in this case?